### ISSUE TITLE
MultivariateNormal random variables need .value() in arguments

### ISSUE THREAD

**Issue by fujiisoup** [blei-lab/edward#492]
Hi

MultivariateNormal random variables (such as MultivariateNormalDiag, except for MultivariateNormalFull)
fails in `ed.utils.copy()`.

The minimum working example is
```python
qf_mu = tf.Variable(np.ones(10), dtype=tf.float32, name='qf_mu')
qf_std = tf.Variable(np.ones(10), dtype=tf.float32, name='qf_std')
qf = MultivariateNormalDiag(mu=qf_mu, diag_stdev=qf_std)
# qf = MultivariateNormalDiag(mu=qf_mu, diag_stdev=qf_std.value()) # <--- This works
ed.utils.copy(qf)
```
The problem seems to be in `tf.contrib.distributions.mvn._MultivariateNormalOperatorPD._assert_valid_mu`,
but I could not figure out how to solve it inside edward.

**Comment by dustinvtran** [comment]
thanks for the note! as you note, looks like it's an issue with the variable being copied vs the variable's _reference_. i'll dig into this.

**Comment by dustinvtran** [comment]
Fixed in #616.
